lib/OpenQA/WebAPI/Plugin/Helpers.pm

```diff
@@ -317,6 +317,22 @@ sub register {
             my ($c, $group) = @_;
             return $c->tag('li', $c->link_to($group->name => $c->url_for('group_overview', groupid => $group->id)));
         });
+
+    $app->helper(
+        comment_icon => sub {
+            my ($c, $jobid, $comment_count) = @_;
+            return '' unless $comment_count;
+
+            return $c->link_to(
+                $c->url_for('test', 'testid' => $jobid) . '#comments' => sub {
```

Comment:
This line causes checkstyle to fail, complaining about 'useless quotes' (christ on a bike, I *hate* this about the perl-verse).

    lib/OpenQA/WebAPI/Plugin/Helpers.pm: Hash key with quotes at line 331, column 37.  Avoid useless quotes.  (Severity: 5)
    lib/OpenQA/WebAPI/Plugin/Helpers.pm: Hash key with quotes at line 331, column 59.  Avoid useless quotes.  (Severity: 5)